sigle/src/ui/IconButton.ts

```diff
@@ -8,7 +8,7 @@ export const IconButton = styled('button', {
 
   '&:disabled': {
     opacity: 0.5,
-    pointerEvents: 'none',
+    cursor: 'not-allowed',
```

Comment:
with pointerEvents set to to none, the tooltip message 'Copied!' fails to show once the copy button is clicked